spec/index.bs

```diff
@@ -3906,9 +3931,8 @@ dictionary GPURasterizationStateDescriptor {
 </script>
 
 <div algorithm>
-    <dfn abstract-op>validating GPURasterizationStateDescriptor</dfn>(|device|, |descriptor|)
-        1. If |device| is lost return `false`.
-        1. If |descriptor|.{{GPURasterizationStateDescriptor/clampDepth}} is `true` and |device|.{{device/[[features]]}}
+    <dfn abstract-op>validating GPURasterizationStateDescriptor</dfn>(|descriptor|, |features|)
+        1. If |descriptor|.{{GPURasterizationStateDescriptor/clampDepth}} is `true` and |features|
             doesn't [=list/contain=] {{GPUFeatureName/"depth-clamping"}}, return `false`.
         1. Return `true`.
```

Comment:
Do we want to validate that clampDepth and the bias stuff is a noop if there is no depth buffer?

Reply:
For `clampDepth` - yes, I think we should be strict about feature checks.
For front face and cull mode, unsure, will add an inline issue.